client/src/components/ProductCard/components/BottomDescription/components/ProductTabs/ProductTabs.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC, useState } from 'react';
+import style from './styles.module.scss';
+import { Tab, Box } from '@mui/material';
+import { TabPanel, TabContext, TabList } from '@mui/lab';
+import product_table from '@assets/product_table.png';
+
+export const ProductTabs: FC = () => {
+  const [value, setValue] = useState('1');
+
+  const handleChange = (event: React.SyntheticEvent, newValue: string) => {
```

Comment:
just (event: MouseEvent

Reply:
better to give a more specific name like

handleTabChange

instead of generic handleChange